BidMachineAdapter/build.gradle.kts

```diff
@@ -73,7 +73,7 @@ dependencies {
     "remoteImplementation"("com.chartboost:chartboost-mediation-sdk:4.0.0")
 
     // Partner SDK
-    implementation("io.bidmachine:ads:2.4.0")
+    implementation("io.bidmachine:ads:2.4.1")
```

Comment:
Don't forget to also change the adapter version.

Reply:
Ah yes.